script/arbitrum-mainnet/deployer.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.22;
+
+import { ConfigHub } from "../../src/ConfigHub.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SetupHelper } from "../setup-helper.sol";
+import { DeploymentHelper } from "../deployment-helper.sol";
+
+library ArbitrumMainnetDeployer {
+    uint constant chainId = 42_161; // id for arbitrum mainnet
+    address constant USDC = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8;
+    address constant USDT = 0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9;
+    address constant WETH = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1;
+    address constant stETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84; // wsteth doesnt have any 0.3% fee with neither USDC nor USDT on arbitrum uniswap
+    address constant weETH = 0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe; // Wrapped weETH on arbitrum doesnt have USDC pools on uniswap
```

Comment:
As comment mentions, this asset is irrelevant for now (doesn't have stable pools)